pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -1171,6 +1173,51 @@ protected void internalSplitNamespaceBundle(AsyncResponse asyncResponse, String
         });
     }
 
+    protected TopicHashPositions internalGetTopicHashPositions(String bundleRange, List<String> topicList) {
```

Comment:
Make it pure async is better to avoid jetty thread block issues.

Reply:
done